devices/stm32f0x0.yaml

```diff
@@ -42,6 +42,7 @@ _include:
  - ./common_patches/merge_USART_CR2_ADDx_fields.yaml
  - ./common_patches/rename_USART_CR1_M0_field.yaml
  - ./common_patches/merge_USART_BRR_fields.yaml
+ - common_patches/tim/tim14_missing_opm.yaml
```

Comment:
```suggestion
```

In RM0360, TIM14 does not have OPM field in CR1.

Reply:
Fixed